src/domain/wallets/tx-history.ts

```diff
@@ -265,13 +304,13 @@ const translateLedgerTxnToWalletTxn = <S extends WalletCurrency>({
   return walletTransaction
 }
 
-const fromLedger = ({
+const fromLedger = <S extends WalletCurrency, T extends DisplayCurrency>({
   ledgerTransactions,
   nonEndUserWalletIds,
 }: {
   ledgerTransactions: LedgerTransaction<WalletCurrency>[]
   nonEndUserWalletIds: WalletId[]
-}): ConfirmedTransactionHistory => {
+}): ConfirmedTransactionHistory<S, T> => {
```

Comment:
`LedgerTransaction<WalletCurrency>[]` should be `LedgerTransaction<S>[]` ?

Reply:
Fixed https://github.com/GaloyMoney/galoy/pull/2434/commits/454c6bdfd80a4a1f321ac9b11a6dad43827ca5d2